package.json

```diff
@@ -65,7 +65,7 @@
     "webpack-dev-server": "^2.8.1"
   },
   "peerDependencies": {
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4"
```

Comment:
What's the associativity here? Does this mean we need **exactly** 16.0.0-0?

For example, does it parse like this

```
(^15.5.4) || 16.0.0-0
```

or like this:

```
^(15.5.4 || 16.0.0-0)
```

Reply:
Ugh good catch, I meant to write: 
```
"react": "^15.5.4 || ^16.0.0-0",
"react-dom": "^15.5.4 || ^16.0.0-0"
```